src/components/shared/partials/FormInputs.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+
+export default function Inputs({ type, minLength, maxLength, regex, className, id, placeholder }) {
```

Comment:
Merge all this into one prop for easier use like [this](https://github.com/dscnitrourkela/project-raisin/blob/2abf016b6937f9eebe8faefc3027c45321d3f460/src/components/shared/partials/Input/index.jsx#L42)

Reply:
done. https://github.com/dscnitrourkela/project-udon/pull/11/commits/3a0e6cfc30790d4efeeddeda42b3b9c5e56f0158